tests/security/tpm2/tpm2_env_setup.pm

```diff
@@ -42,22 +53,22 @@ ConditionPathExistsGlob=
 EOF
         )\" > /etc/systemd/system/tpm2-abrmd.service.d/emulator.conf"
     );
-    assert_script_run "systemctl daemon-reload";
-    assert_script_run "systemctl enable tpm2-abrmd";
+    assert_script_run("systemctl daemon-reload");
+    assert_script_run("systemctl enable tpm2-abrmd");
 
     # Reboot the node to make the changes take effect
-    power_action('reboot', textmode => 1);
+    power_action("reboot", textmode => 1);
```

Comment:
Is that reboot actually necessary?

Reply:
> What is the `tss` user used for? Does the test also work if you just remove the user creation altogether? It's already created by installation of `tpm2-0-tss`.

Good question, here comes the comments from developer:

"first I'd like to suggest not to run the emulator and the abrmd as root.
It's a bad practice and the test coverage will be better when the abrmd runs as the tss user as it is supposed to do"

Reply:
> 

I don't have to to that, let me fix it as well